### ISSUE TITLE
Question about the paper

### ISSUE THREAD

**Issue by Mathilda88** [Ha0Tang/XingVTON#4]
Thanks for the great job. I have a question about the ablation study. I'm a little bit confused if for the ablation you have trained the models from scratch (without the ablated part) or just used the pretrained model and set the weight values of that loss to zero? 

Thank you in advance.

**Comment by Amazingren** [comment]
> Thanks for the great job. I have a question about the ablation study. I'm a little bit confused if for the ablation you have trained the models from scratch (without the ablated part) or just used the pretrained model and set the weight values of that loss to zero?
> 
> Thank you in advance.

Thanks for your interest, we train our model from scratch for ablation study for XingVTON parts. I guess it would be more clear about the experiment when our overall paper is released in maybe a few days later.

**Comment by Mathilda88** [comment]
thank you so much for your response. So that means we have to put the weight of the loss to zero (for the ablated part) or change the structure of our code to disentangle the part?

**Comment by Amazingren** [comment]
> thank you so much for your response. So that means we have to put the weight of the loss to zero (for the ablated part) or change the structure of our code to disentangle the part?

Hi @Eliot04,

For XingVTON in our work, we share the same loss item as the previous work [CP-VTON+](https://github.com/minar09/cp-vton-plus) did.   If you are interested in the loss parts, maybe you can turn to CV-VTON+ for some help. 

By the way, since we didn't release our paper at the moment, so I am not very sure what you mean about the ablation study and what part you want to use in our codebase.  
But you will see it soon, we can have a discussion by then.

Best

**Comment by Mathilda88** [comment]
Thank you for your response, look forward to having it.

**Comment by Gzzgz** [comment]
I can't find the paper.
